### ISSUE TITLE
Appllication load balancer help text incorrect

### ISSUE THREAD

**Issue by mrndev** [ionos-cloud/ionosctl#263]
## Description
Here is the help text output for `ionosctl applicationloadbalancer rule httprule add --help` - see the bold default faules
...
  -C, --condition string                    Matching rule for the HTTP rule condition attribute; mandatory for HEADER, PATH, QUERY, METHOD, HOST, and COOKIE types; must be null when type is SOURCE_IP. (default "**STARTS_WITH**")
  -K, --condition-key string                Must be null when type is PATH, METHOD, HOST, or SOURCE_IP. Key can only be set when type is COOKIES, HEADER, or QUERY. (default "**forward-at**")
  -T, --condition-type string               Type of the HTTP rule condition. (default "HEADER")
  -V, --condition-value string              Mandatory for conditions CONTAINS, EQUALS, MATCHES, STARTS_WITH, ENDS_WITH; must be null when condition is EXISTS; should be a valid CIDR if provided and if type is SOURCE_IP. (default "**Friday**")
....

## Expected behavior
Correct default values in the help text

## Environment

Ionosctl version: 
ionosctl --version
ionosctl version v6.5.2

## How to Reproduce

ionosctl applicationloadbalancer rule httprule add --help

**Comment by avirtopeanu-ionos** [comment]
For `--condition` and `--condition-key`, can you please elaborate what you think is wrong?

`definitions.ApplicationLoadBalancerHttpRuleCondition.properties.condition`:
```
{"type": "string","example": "STARTS_WITH","description": "The matching rule for the HTTP rule condition attribute; this parameter is mandatory for 'HEADER', 'PATH', 'QUERY', 'METHOD', 'HOST', and 'COOKIE' types. It must be 'null' if the type is 'SOURCE_IP'.","enum": ["EXISTS","CONTAINS","EQUALS","MATCHES","STARTS_WITH","ENDS_WITH"]}
```

`definitions.ApplicationLoadBalancerHttpRuleCondition.properties.key`:
```
{"type": "string","example": "forward-at","description": "The key can only be set when the HTTP rule condition type is 'COOKIES', 'HEADER', or 'QUERY'. For the type 'PATH', 'METHOD', 'HOST', or 'SOURCE_IP' the value must be 'null'."}
```

surprisingly, Friday is also the `example` given in the swagger spec for `definitions.ApplicationLoadBalancerHttpRuleCondition.properties.value`:
```
{"type": "string","example": "Friday","description": "This parameter is mandatory for the conditions 'CONTAINS', 'EQUALS', 'MATCHES', 'STARTS_WITH', 'ENDS_WITH', or if the type is 'SOURCE_IP'. Specify a valid CIDR. If the condition is 'EXISTS', the value must be 'null'."}
```

what are the correct swagger spec values?

**Comment by avirtopeanu-ionos** [comment]
Hello, thank you opening this issue. This has been fixed in [release v6.6.2](https://github.com/ionos-cloud/ionosctl/releases/tag/v6.6.2)

Have a nice day!
